apisix/cli/ops.lua

```diff
@@ -797,9 +802,10 @@ end
 local function test(env, backup_ngx_conf)
     -- backup nginx.conf
     local ngx_conf_path = env.apisix_home .. "/conf/nginx.conf"
-    local ngx_conf_exist = util.is_file_exist(ngx_conf_path)
```

Comment:
Look like we can remove `is_file_exist` in `util` now?

Reply:
Remove it after `cert_path` checking with `pl_path.exist`. Had done.